nomad/leader.go

```diff
@@ -990,6 +993,19 @@ func (s *Server) publishJobStatusMetrics(stopCh chan struct{}) {
 	}
 }
 
+func (s *Server) publishEventsForSinks(stopCh chan struct{}) {
```

Comment:
This should be removed, in earlier iterations I created the event sink manager here instead of during server setup, now the manager is initialized with the server's shutdown context, so this isn't needed.